src/towncrier/test/test_check.py

```diff
@@ -166,6 +166,71 @@ def test_fragment_missing(self):
                 result.output.endswith("No new newsfragments found on this branch.\n")
             )
 
+    def test_fragment_exists_but_not_in_check(self):
+        runner = CliRunner()
+
+        with runner.isolated_filesystem():
+            create_project(
+                "pyproject.toml",
+                main_branch="master",
+                extra_config="[[tool.towncrier.type]]\n"
+                'directory = "feature"\n'
+                'name = "Features"\n'
+                "showcontent = true\n"
+                "[[tool.towncrier.type]]\n"
+                'directory = "auto"\n'
+                'name = "Automatic"\n'
```

Comment:
very minor comment... just to make it clear that this is the scope/targe of this test.

```suggestion
                'directory = "sut"\n'
                'name = "System Under Test"\n'
```